--- /home/charl/defiformal-wt-sprint11-grok-gpt6-20260907/review/semantic-kernel/sprint11/implementation/causal-generic-r1/hashes/private-lean-sources.pre-sync.sha256	2026-09-07 21:43:34.152046478 -0600
+++ /home/charl/defiformal-wt-sprint11-grok-gpt6-20260907/review/semantic-kernel/sprint11/implementation/causal-generic-r1/hashes/worktree-lean-sources.pre-sync.sha256	2026-09-07 21:43:33.880046524 -0600
@@ -105,16 +105,16 @@
 575ebbbfa8f868af557edb1cd3f1c1c284caa28148bc08b5471f53ee749a6d90  ./DefiKernel/Nary/Audit.lean
 9c78ea1e43e059c8e1538937b66c941c71018fcfd86b35e81f424d91a06816d6  ./DefiKernel/Nary/CausalRuntime.lean
 3563dbb15ae4af53a434cdf2080118b29e65dc3a148c78e4cb2e7336c0649847  ./DefiKernel/Nary/Completion.lean
-af71c740fdb698b204592c88003fdfa1573b8dee1c900e14f8cfd956d17b5726  ./DefiKernel/Nary/Examples.lean
+1f844aad7c8912148d3a0660a76c670f91b0c262dc1e847c558e3ea964b2b8a4  ./DefiKernel/Nary/Examples.lean
 05bea52c31d5e2841f4d2834307c122db5b10e751984fa50f146ef526d34aab9  ./DefiKernel/Nary/Execution.lean
 82a0cf1ad4ba38d8c0be3b0c84b92ec54388e582407c510839ad85c7914a93fe  ./DefiKernel/Nary/Interference.lean
 6006dd4610e25603606b86c46ab00fc626680d02fd4604a65af91b9a848111ed  ./DefiKernel/Nary/LocalOrder.lean
 f64500ddb68bd3ee5fa99dca866e038e940eb31a8b94042c2579d1f71e187751  ./DefiKernel/Nary/Observation.lean
-dffe6f3159a172c37c2efcdb13f8113fe4147a8f8714a7056454eb091c75fd52  ./DefiKernel/Nary/Preservation.lean
+dcecd9cd2567260b52b25dc9ca682ea3279f2b745e3e02cf44e70dc4cef15fe9  ./DefiKernel/Nary/Preservation.lean
 24eb7a254f8ae4f2664cc4089ec4ab5e384a87a11a4fd648b7cf5edc24cb7938  ./DefiKernel/Nary/Schedule.lean
 8c1969b069ae893a350679963c48f24fac9bf3325645c2a5327317c1360baa23  ./DefiKernel/Nary/Soundness.lean
-bab16cab97658c5070e1f9197b258192bacf0ba10e6c7f6f2c016580fa96b49a  ./DefiKernel/Nary/Tests.lean
-e3be28916e3ba59211d8ef2e7a2c2bdfc81f09e8af9cc040f49481e8c0c8214a  ./DefiKernel/Nary/Trace.lean
+27576cb692b8befe6451910b076f8f06f870d76594d28681f15e44d5fbd0a05d  ./DefiKernel/Nary/Tests.lean
+10d0d4cb8f9419aa3a7dfb4d0a16204fd7e96d2051d8d9eabbaba39b63c7e94c  ./DefiKernel/Nary/Trace.lean
 d6cb7e1c7b948b4404007a50ca30bf5bcf5883bd77c6ce700b229884d4e094b9  ./DefiKernel/Parallel/Audit.lean
 c85f69f1bfad39700096c95dbd1450e65eb5f102d4b08a80054f45edf96c52ef  ./DefiKernel/Parallel/Commutation.lean
 4459fa2b4a4f1f695d3856cb67a46a7c9df86a90f924be8bd26f55e99ba54243  ./DefiKernel/Parallel/Compatibility.lean
